src/js/camresolutionstest.js

```diff
@@ -144,9 +144,9 @@ CamResolutionsTest.prototype = {
     video.setAttribute('muted', '');
     video.width = resolution[0];
     video.height = resolution[1];
-    attachMediaStream(video, stream);
+    video.srcObject = stream;
```

Comment:
Can you make all the calls of this project to attachMediaStream to go via the same mechanism.

Either all of them use: "video.srcObject = stream" or all of them use "adapter.browserShim.attachMediaStream(video, stream)".